deps/build.jl

```diff
@@ -64,13 +59,21 @@ ENV2["CFLAGS"] = cflags
 
 provides(SimpleBuild,
     (@build_steps begin
-        GetSources(scs)
+        GetSources(direct)
         CreateDirectory(joinpath(prefix, "lib"))
-        FileRule(joinpath(prefix, "lib", libname), @build_steps begin
+
+        FileRule(joinpath(prefix, "lib", "libscsdir.$(Libdl.dlext)"), @build_steps begin
             ChangeDirectory(srcdir)
-            setenv(`make BLASLDFLAGS= out/$libname`, ENV2)
-            `mv out/$libname $prefix/lib`
+            setenv(`make BLASLDFLAGS= out/libscsdir.$(Libdl.dlext)`, ENV2)
+            `mv out/libscsdir.so $prefix/lib`
```

Comment:
The build is failing on OS X because of the `.so` extension here: https://travis-ci.org/JuliaOpt/SCS.jl/jobs/313571982#L360
Please try to get OS X passing.